examples/accessible_text/main.rs

```diff
@@ -0,0 +1,24 @@
+mod text_view;
+
+use gtk::{glib, prelude::*};
+use text_view::AccessibleTextView;
+
+fn main() -> glib::ExitCode {
+    let application = gtk::Application::builder()
+        .application_id("com.github.gtk-rs.examples.accessible_text")
+        .build();
+    application.connect_activate(build_ui);
+    application.run()
+}
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+
+    window.set_title(Some("Accessible Text Example"));
+    window.set_default_size(260, 140);
+
+    let text_view: AccessibleTextView = glib::Object::new();
```

Comment:
```suggestion
    let text_view = glib::Object::new::<AccessibleTextView>();
```
turbo fish for the win